taxcalc/records.py

```diff
@@ -730,22 +731,29 @@ def _impute_variables(self):
         # std_2009 = np.array([6100, 12200, 6100, 8950, 12200, 6100, 1000])
         # Additional standard deduction for aged 2009
         std_aged_2009 = np.array([1400., 1100.])
+        # std_aged_2009 = np.array([1500., 1200.])
         # impute number of taxpayers
         self._txpyers = np.where(np.logical_or(self.MARS == 2,
                                                np.logical_or(self.MARS == 3,
                                                              self.MARS == 6)),
                                  2., 1.)
         # impute number of extra standard deductions for aged
-        self._numextra = np.where(np.logical_and(self.FDED == 2, self.e04470 >
-                                                 std_2009[self.MARS - 1]),
-                                  np.where(
-                                      np.logical_and(self.MARS != 2,
-                                                     self.MARS != 3),
-                                      (self.e04470 - std_2009[self.MARS - 1]) /
-                                      std_aged_2009[0],
-                                      (self.e04470 - std_2009[self.MARS - 1]) /
-                                      std_aged_2009[1]),
-                                  np.where(self.e02400 > 0, self._txpyers, 0))
+        self._numextra = np.where(np.logical_or(self.AGERANGE >= 6,
```

Comment:
Variable `AGERANGE` being incorporated to get more accurate estimation of `_numextra`.